addons/account/models/account_move.py

```diff
@@ -3982,6 +3982,21 @@ def _notify_get_recipients_groups(self, msg_vals=None):
     def _get_report_base_filename(self):
         return self._get_move_display_name()
 
+    def _get_report_attachment_filename(self):
+        self.ensure_one()
+        if self.state == 'posted':
+            name = self.name or _('INV')
+        else:
+            name = self._get_report_base_filename()
+        return name.replace('/', '_') + '.pdf'
```

Comment:
Nitpick, feel free to ignore
```suggestion
        return f"{name.replace('/', '_')}.pdf"
```